src/RFlow/Barakhadi.jsx

```
@@ -4180,6 +4180,8 @@ const Barakhadi = ({
   customWords, // Array of words to filter (e.g., ["నది", "చేప", "చీర"] for Telugu F2)
 }) => {
   steps = 1;
+  const isMobile = useMediaQuery(theme.breakpoints.down("sm"));
+  const isTablet = useMediaQuery(theme.breakpoints.between("sm", "md"));
```

ESLint

[error] 'isTablet' is assigned a value but never used.

(no-unused-vars)